.github/workflows/deploy.yaml

```diff
@@ -91,3 +91,23 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  create_draft_release:
+    if: github.ref_type == 'tag'
+    permissions:
+      contents: write
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
```suggestion
      GITHUB_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
```

Reply:
oops:

<img width="422" alt="image" src="https://user-images.githubusercontent.com/76071/189364989-5ed5972a-5209-4de6-86cd-b351b2d6dcdd.png">

I'll call this GH_RELEASE_TOKEN.

Reply:
Ah sure, sounds good!